tools/Dockerfile

```
@@ -78,7 +78,8 @@ WORKDIR $APP_HOME
 ENV PATH $PATH:$APP_HOME
 
 # kubectl
-ADD https://storage.googleapis.com/kubernetes-release/release/v${KUBECTL_VERSION}/bin/linux/amd64/kubectl kubectl
+ADD https://cdn.dl.k8s.io/release/v${KUBECTL_VERSION}/kubernetes-client-linux-amd64.tar.gz /tmp
+RUN tar -zxvf /tmp/kubernetes-client-linux-amd64.tar.gz -C /tmp && mv /tmp/kubernetes/client/bin/kubectl kubectl && rm -rf /tmp/kubernetes
```

Can we add also the checksum for the kubectl binary? Also the official kubernetes [documantation](https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/#install-kubectl-binary-with-curl-on-linux) points to the below link.

```suggestion
ADD --checksum=sha256:98ea4a13895e54ba24f57e0d369ff6be0d3906895305d5390197069b1da12ae2 https://dl.k8s.io/release/v$KUBECTL_VERSION/bin/linux/amd64/kubectl .
```